src/nomadic/dashboard/main.py

```diff
@@ -14,6 +15,13 @@ def find_metadata(input_dir: str) -> MetadataTableParser:
     """
 
     metadata_dir = os.path.join(input_dir, "metadata")
+
+    # first check if the file with standard name exists
+    standard_path = os.path.join(metadata_dir, "sample-info.csv")
```

Comment:
Let's think exactly what we want the standard name to be. We should have a consistent vocabulary for this file across Nomadic and name accordingly.

Reply:
I took sample-info because I think it is what you used before, no?
For me this file contains the samples, so `samples.csv`, seems the most concise.